code/modules/organs/internal/muscles.dm

```diff
@@ -16,6 +16,8 @@
 	icon_state = "robotic_muscle"
 	desc = "Expand and contract"
 	nature = MODIFICATION_SILICON
+	organ_efficiency = list(OP_MUSCLE = 200)
```

Comment:
Why are we making robots and the robotic printer better then the upgraded human ones?